test/linalg/symmetric.jl

```diff
@@ -30,8 +30,7 @@ end
     n = 10
     areal = randn(n,n)/2
     aimg  = randn(n,n)/2
-    @testset "symmetric eigendecomposition with element type $(eltya)" for eltya in
-            (Float32, Float64, Complex64, Complex128, BigFloat, Int)
+    @testset for eltya in (Float32, Float64, Complex64, Complex128, BigFloat, Int)
         a = eltya == Int ? rand(1:7, n, n) : convert(Matrix{eltya}, eltya <: Complex ? complex.(areal, aimg) : areal)
         asym = a'+a                 # symmetric indefinite
```

Comment:
I think that's a good idea. I know for a while we had some methods that just wouldn't work with symmetric complex matrices, hopefully that's no longer the case.